ffi/record_meta.c

```diff
@@ -8,10 +8,12 @@
 #define EXPORT
 #endif
 
-typedef struct meta_t {
+/* TODO: tis is replicated in ffi_platypus.h, which is bad */
+typedef struct _ffi_pl_record_meta_t {
```

Comment:
This struct is replicated in two places because of the way the build system is currently working.  Lets define it in just one place.